wire-ios-request-strategy/Sources/Synchronization/Decoding/EventDecoder.swift

```diff
@@ -98,112 +115,117 @@ extension EventDecoder {
     /// - Parameters:
     ///   - encryptionKeys: Keys to be used to decrypt events.
     ///   - block: Event consume block which is called once for every stored event.
+
     public func processStoredEvents(with encryptionKeys: EncryptionKeys? = nil, _ block: ConsumeBlock) {
         process(with: encryptionKeys, block, firstCall: true)
     }
 
     /// Decrypts and stores the decrypted events as `StoreUpdateEvent` in the event database.
     /// The encryption context is only closed after the events have been stored, which ensures
     /// they can be decrypted again in case of a crash.
-    /// - parameter events The new events that should be decrypted and stored in the database.
-    /// - parameter startingAtIndex The startIndex to be used for the incrementing sortIndex of the stored events.
-    /// - Returns: Decrypted events
-    fileprivate func decryptAndStoreEvents(_ events: [ZMUpdateEvent], startingAtIndex startIndex: Int64) -> [ZMUpdateEvent] {
-        let account = Account(userName: "", userIdentifier: ZMUser.selfUser(in: self.syncMOC).remoteIdentifier)
-        let publicKey = try? EncryptionKeys.publicKey(for: account)
+    ///
+    /// - Parameters:
+    ///   - events The new events that should be decrypted and stored in the database.
+    ///   - startingAtIndex The startIndex to be used for the incrementing sortIndex of the stored events.
+    ///
+    /// - Returns: Decrypted events.
+
+    private func decryptAndStoreEvents(
```

Comment:
does Xcode pick the documentation if there is a space here?

Reply:
Yes it does. I usually keep a space to create a bit more vertical separation to make it a bit easier to read, but I'm happy to change it.

Reply:
if it does pick it I don't mind, we might want to add a swiftlint rule to enforce it so it's consistent